install

```diff
@@ -41,6 +41,18 @@ _() {
       echo 'Error: Either yarn or npm must be installed (https://www.npmjs.com/)' >&2
       exit 1
     fi
+    if ! [ -x "$(command -v terraform)" ]; then
```

Comment:
Are we going to require all these as hard bedrock dependencies? I kind of think there will be pushback to this. Nearly everyone has git and node/npm installed but most will not have any of these, and I kind of doubt we want to make them install everything to try out bedrock quickly. Also what if they want to deploy to another platform, etc etc

Reply:
Good point. I'll move this check to when you actually try to run a `cloud` command.

Reply:
also maybe a note when they run `bedrock cloud` command? (if it's not there already) ... I think you can update the `description` field in `commands.json`.

Reply:
I mean, `cloud provision` command.